src/main/java/n3/arrays/ArrayTask16.java

```diff
@@ -0,0 +1,23 @@
+package n3.arrays;
+
+import java.util.Arrays;
+
+/*
+- написати програму яка друкує всі елементи із значень матриці ( двовимірного масиву int[][] arr = ... )
+ */
+
+public class ArrayTask16 {
+    public static void main(String[] args) {
+        int i, k;
+        int[][] arr = {{5, 6, 1, 3}, {3, 4, 2, 1}, {1, 2, 2, 2}};
```

Comment:
Use input :
int[][] arr = {{5, 6, 1, 3}, {3, 4, 2, 1}, {1, 2, 2, 2}, {3, 4, 5, 6}};

and then fix a problem.

Reply:
fixed